src/components/UserComponents/UserDisplay.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { UserSchema } from "../../schemas/UserSchema";
+import { SizeType } from "../../constants";
+import { Card, CardBody, Avatar, Text } from "@chakra-ui/react";
+import {
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+  PopoverArrow,
+  PopoverCloseButton,
+} from "@chakra-ui/react";
+
+interface AuthorProps {
+  user: UserSchema;
+  size: SizeType;
+}
+
+export default function AuthorComponent({ user, size }: AuthorProps) {
```

Comment:
⛏️ Convention is to name this after the file

Reply:
Is it still convention if it's not a default export? Removed the default export so other UserDisplay components can be exported from that file in the future.